src/helics/shared_api_library/FederateExport.cpp

```diff
@@ -1224,3 +1224,52 @@ void helicsFederateLogLevelMessage(helics_federate fed, int loglevel, const char
     }
     fedObj->logMessage(loglevel, AS_STRING(logmessage));
 }
+
+void helicsFederateSendCommand(helics_federate fed, const char* target, const char* command, helics_error* err)
+{
+    auto* fedObj = getFed(fed, err);
+    if (fedObj == nullptr) {
+        return;
+    }
+    fedObj->sendCommand(AS_STRING(target), AS_STRING(command));
+}
+
+const char* helicsFederateGetCommand(helics_federate fed, helics_error* err)
+{
+    auto* fedObj = helics::getFedObject(fed, err);
```

Comment:
Just confirming because I do not have the code open at the moment:
getFedObject returns an access pointer to an object persistent in memory elsewhere, correct? (verifying that the use of `c_str()` below will be an access into a persistent data point and not a temporary).

Reply:
Yeah the actual command and source get copied to a value in the FederateObject,  So it is stored until the next command is retrieved.